RevenueCatUI/CustomerCenter/Extensions/CustomerCenterConfigDataSupport+URL.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CustomerCenterConfigDataSupport+Button.swift
+//
+//  Created by Facundo Menzella on 2/4/25.
+
+import Foundation
+import RevenueCat
+
+#if os(iOS)
+
+extension CustomerCenterConfigData.Support {
+
+    @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
```

Comment:
Not a big deal, but I still wonder why we have
```
    @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
``` 
and then 
```
    @available(macOS, unavailable)
    @available(tvOS, unavailable)
    @available(watchOS, unavailable)
```

I guess we'll need to make this consistent eventually 😄

Reply:
Oh no wait, I took this from `URLUtilities.createMailURLIfPossible`. It was throwing errors otherwise